src/main/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaDataResultSets.java

```diff
@@ -1754,4 +1755,319 @@ static ImmutableList<JdbcColumnMetaData> buildImportedKeysColumnMetaData(
+                            true, //signed,
+                            64, //displaySize,
+                            "MINIMUM_SCALE", //label,
+                            "MINIMUM_SCALE", //columnName,
+                            null, //schemaName,
+                            0, //precision,
+                            0, //scale,
+                            Types.SMALLINT, //type.id,
+                            JdbcType.SMALLINT.name(), //type.name,
+                            short.class.getName()) //columnClassName
+                    )
+                    .add(new JdbcColumnMetaData(
+                            ordinal++,
+                            false, //caseSensitive,
+                            ResultSetMetaData.columnNoNulls, //nullable,
+                            true, //signed,
+                            64, //displaySize,
+                            "MAXIMUM_SCALE", //label,
+                            "MAXIMUM_SCALE", //columnName,
+                            null, //schemaName,
+                            0, //precision,
+                            0, //scale,
+                            Types.SMALLINT, //type.id,
+                            JdbcType.SMALLINT.name(), //type.name,
+                            short.class.getName()) //columnClassName
+                    )
+                    .add(new JdbcColumnMetaData(
+                            ordinal++,
+                            false, //caseSensitive,
+                            ResultSetMetaData.columnNoNulls, //nullable,
+                            true, //signed,
+                            64, //displaySize,
+                            "SQL_DATA_TYPE", //label,
+                            "SQL_DATA_TYPE", //columnName,
+                            null, //schemaName,
+                            0, //precision,
+                            0, //scale,
+                            Types.INTEGER, //type.id,
+                            JdbcType.INTEGER.name(), //type.name,
+                            int.class.getName()) //columnClassName
+                    )
+                    .add(new JdbcColumnMetaData(
+                            ordinal++,
+                            false, //caseSensitive,
+                            ResultSetMetaData.columnNoNulls, //nullable,
+                            true, //signed,
+                            64, //displaySize,
+                            "SQL_DATETIME_SUB", //label,
+                            "SQL_DATETIME_SUB", //columnName,
+                            null, //schemaName,
+                            0, //precision,
+                            0, //scale,
+                            Types.INTEGER, //type.id,
+                            JdbcType.INTEGER.name(), //type.name,
+                            int.class.getName()) //columnClassName
+                    )
+                    .add(new JdbcColumnMetaData(
+                            ordinal,
```

Comment:
ordinal not incremented here? should it be `ordinal++`?

Reply:
@alinaliBQ - This is correct. I will add comment like in previous occasions.

Reply:
Done.